.changelog/12366.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:enhancement
+api: URL-encode/decode resource names for all HTTP API endpoints (added in Consul 1.12.0 via #11335, #11957, #12103, #12190, #12297)

```

Comment:
@kisunji : how should we handle this changelog entry? The functionality was released in 1.12.0 in the aforementioned PR, but using the normal merge process, this would actually show up in the 1.12.1 section of the changelog (AFAIK).

We're adding an entry for something that already exists (but isn't documented yet), rather than adding/releasing new functionality with this PR.

Reply:
We agreed I'd just remove the changelog entry from the PR